benchmarks/templates/benchmarks/tutorial_deepdive_1.html

```diff
@@ -66,23 +88,24 @@ <h3 class="benefits_heading is-size-3-mobile">Step 1: Install Necessary Packages
                 <div class="column is-three-quarters has-text-centered">
                     <h3 class="benefits_heading is-size-3-mobile">Step 2: Download the starter submission Zip folder: </h3>
                        <p> Brain-Score allows users to submit plugins in two ways: directly to the site, via a Zip file
-                           upload, or through a Github PR. In this deepdive, you will upload a sample Zip project to
-                           the site in order to learn how the submission system works. In Deepdive 3, you will submit a
-                           model via Github Pull Request. The sample Zip is a stock version of Resnet-50 pretrained on
-                           imagenet, and you should get a score back within 24 hours.
-                           You can download the sample zip file here.
+                           upload, or through a Github PR. In this deepdive, you will prepare a sample Zip project to
+                           upload (but will not actually submit it), in order to learn the process of submitting.
+                           In Deepdive 3, you will prepare to submit a model via Github Pull Request. The sample Zip
+                           is a stock version of resnet50 pretrained on imagenet. You can download the sample zip file
+                           <a href="{% static "/benchmarks/packaging_tutorial.zip" %}" download="packaging_tutorial.zip">here</a>.
                        </p>
                 </div>
             </div>
 
            <div class="columns is-centered">
                 <div class="column is-three-quarters has-text-centered">
                     <h3 class="benefits_heading is-size-3-mobile">Step 3: Explore the starter submission file</h3>
-                       <p> After unzipping the model.zip, you will see that it contains a folder called
-                           resnet-50_deepdive-2, which in turn contains 4 other directories.
-                           The overall structure of the submission package is below. *Note*: this would also be
-                           the same overall structure as your submission package, if you are submitting any other
-                           plugins.
+                       <p> After unzipping the model.zip, you will see that it contains a folder called models
+                           (the plugin), which in turn contains the model itself, named resnet50_tutorial.
+                           This folder also contains 4 other Python files. The overall structure of the submission
+                           package is below. *Note*: this would also be the same overall structure as your submission
+                           package, if you are submitting any other plugins. Feel free to base your submission off of
+                           this package, just do not submit the resnet50_tutorial model.
```

Comment:
it's weird to me that we keep saying they aren't allowed to submit the tutorial model. We should have guardrails in place if they do right? I would mention it once in the submit section, but not talk about it so much otherwise. Don't feel strongly though, so feel free to keep if there are good reasons